recipes/uchardet/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(uchardet REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} uchardet)
```

Looks like [they export the `uchardet::uchardet` target](https://github.com/freedesktop/uchardet/blob/b3b2bd2721e389e8392d1764ecd322ffe98ac0d9/CMakeLists.txt#L81).
```suggestion
target_link_libraries(${PROJECT_NAME} uchardet::uchardet)
```